internal/pkg/cli/svc_init_test.go

```diff
@@ -201,6 +201,18 @@ func TestSvcInitOpts_Ask(t *testing.T) {
 			mockDockerfile: func(m *mocks.MockdockerfileParser) {},
 			wantedErr:      fmt.Errorf("some error"),
 		},
+		"skip asking for port for backend service": {
+			inSvcType:        "Backend Service",
+			inSvcName:        wantedSvcName,
+			inDockerfilePath: wantedDockerfilePath,
+
+			mockPrompt: func(m *mocks.Mockprompter) {},
+			mockDockerfile: func(m *mocks.MockdockerfileParser) {
+				m.EXPECT().GetExposedPorts().Return([]uint16{}, errors.New("no expose"))
```

Comment:
an alternative way of writing this:
```suggestion
				m.EXPECT().GetExposedPorts().Times(0)
```